<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-GGIOLCA8" failures="1" tests="5" name="Test" time="24.565" errors="0" timestamp="2025-01-29T09:02:57 IST">
  <testcase classname="testCases.TC003_LoginTest_DDT" name="verify_loginDDT" time="2.491"/>
  <testcase classname="testCases.TC003_LoginTest_DDT" name="verify_loginDDT" time="2.061"/>
  <testcase classname="testCases.TC003_LoginTest_DDT" name="verify_loginDDT" time="2.068">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginTest_DDT.verify_loginDDT(TC003_LoginTest_DDT.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginTest_DDT" name="verify_loginDDT" time="11.266"/>
  <testcase classname="testCases.TC003_LoginTest_DDT" name="verify_loginDDT" time="2.177"/>
</testsuite> <!-- Test -->
